server/sonar-server/src/test/java/org/sonar/server/user/ws/ChangePasswordActionTest.java

```diff
@@ -51,7 +56,9 @@
 import static com.google.common.collect.Lists.newArrayList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
I'd prefer no to use the Mockito runner avec declare explicitly Mockito.mock(…) the mocks. I think it's the direction we're taking on the platform